clustered/server/src/main/java/org/ehcache/clustered/server/EhcacheStateServiceImpl.java

```diff
@@ -109,18 +109,26 @@ public ServerStoreImpl getStore(String name) {
     return Collections.unmodifiableSet(dedicatedResourcePools.keySet());
   }
 
-  String getDefaultServerResource() {
+  public String getDefaultServerResource() {
     return this.defaultServerResource;
   }
 
-  public void validate(ValidateStoreManager message) throws ClusterException {
+  @Override
+  public Map<String, ServerSideConfiguration.Pool> getSharedResourcePools() {
+    Map<String, ServerSideConfiguration.Pool> sharedPools = new HashMap<String, ServerSideConfiguration.Pool>();
+    for (Map.Entry<String, ResourcePageSource> entry : sharedResourcePools.entrySet()) {
+      sharedPools.put(entry.getKey(), entry.getValue().getPool());
+    }
+    return sharedPools;
```

Comment:
We have moved this to Java 8:
`return sharedResourcePools.entrySet().stream().collect(toMap(Map.Entry::getKey, e -> e.getValue().getPool()))`